### PR TITLE
Viewer for the tiling xrays quadtree (Server).

### LINKED ISSUE #90: Generate tiling x-rays out of an octree

### PR DESCRIPTION
**Pull request by SirVer**
Fixes #90.

### CODE REVIEW

**Comment by SirVer** [comment]
See #98 for the client. #98 must be merged first.

File: xray/src/bin/web_viewer.rs [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+use quadtree::{ChildIndex, Node};
+use router::Router;
+use std::fs;
+use std::io::Read;
+use std::path::PathBuf;
+use std::sync::Arc;
+use urlencoded::UrlEncodedQuery;
+use xray::Meta;
+
+const INDEX_HTML: &'static str = include_str!("../../client/index.html");
```
Comment (thomasschiwietz) [inline xray/src/bin/web_viewer.rs]: The octree web_viewer and the quadtree web_viewer put their in the same location.

Comment (SirVer) [reply xray/src/bin/web_viewer.rs]: Actually that wasn't the case. We misread the code yesterday.

**Comment by SirVer** [comment]
@thomasschiwietz PTAL

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

**Pull request merged**